### PR TITLE
ELEMENTS-619: allow to initialize through seleted-item(s) + fix multi…

### PR DESCRIPTION
**Pull request by guirenard**
… complex editing with document field + clean up

### CODE REVIEW

File: nuxeo-data-table/iron-data-table.html [hunk 1]
```diff
@@ -744,22 +744,40 @@ <h2>[[i18n('command.add')]]</h2>
...
           }
           this.notifyResize();
           this.$.dialog.close();
         }
       },
 
+      _deepCopy: function(obj) {
+        var cache = [];
+        var result = JSON.parse(JSON.stringify(obj, function(key, value) {
+            if (typeof value === 'object' && value !== null) {
```
Comment (afjusto) [inline nuxeo-data-table/iron-data-table.html]: this is not formatted with 2 spaces, right?

Comment (guirenard) [reply nuxeo-data-table/iron-data-table.html]: not really :)

**Comment by afjusto** [comment]
👍

**Comment by adilioaraujo** [comment]
:+1:

**Pull request merged**